### DISCUSSION TITLE
How to reduce  one disk per server forever

### DISCUSSION THREAD

**Discussion by muziling** [minio/minio#20206] [Q&A] [answered]
My current minio conf:   MINIO_VOLUMES="http://minio{1...2}:9000/data{1...4}/minio"
How to safely reduce disk from  data{1...4} to data{1...3}?

Is this step? 
1、umount two server's disk4
2、modify MINIO_VOLUMES  to MINIO_VOLUMES="http://minio{1...2}:9000/data{1...3}/minio"
2、mc restart minio

**Comment by harshavardhana** [comment, marked as answer]
You cannot
